dcrpool.go

```diff
@@ -26,6 +26,11 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
+const (
+	// dbBackupFile is the database backup file name.
+	dbBackupFile = "backup.kv"
```

Comment:
Since backups are reserved for just th bbolt db implementation should this not be defined there?

Reply:
Fair point yes. My instinct was to hoist it up the hard-coded string so it could eventually be moved into the config file, but in this case I guess there isnt much point in that. Restored